.github/dependabot.yaml

```diff
@@ -1,8 +0,0 @@
-version: 2
```

Comment:
Why does this show up as deleted, and then there is a new one with the exact same name? This is strange...

Reply:
You added two files here earlier. dependabot.yaml and dependabot.yml. I thought that was a mistake and we just need the .yml file ?

Reply:
Wow, that's right! I didn't even notice it.